fedgov-cv-mapencoder/src/main/java/gov/usdot/cv/mapencoder/LaneDataAttribute.java

```diff
@@ -24,18 +24,28 @@ public class LaneDataAttribute {
     private int laneCrownPointLeft;
     private int laneAngle;
     private SpeedLimitList speedLimits;
+    private boolean speedLimitsExist;
+
+    //enum choices
+    public static final int PATHENDPOINTANGLE = 1;
+    public static final int LANECROWNPOINTCENTER = 2;
+    public static final int LANECROWNPOINTRIGHT = 3;
+    public static final int LANECROWNPOINTLEFT = 4;
+    public static final int LANEANGLE = 5;
+    public static final int SPEEDLIMITS = 6;
 
     public LaneDataAttribute() {
     }
 
-    public LaneDataAttribute(int choice, int pathEndPointAngle, int laneCrownPointCenter, int laneCrownPointRight, int laneCrownPointLeft, int laneAngle, SpeedLimitList speedLimits) {
+    public LaneDataAttribute(int choice, int pathEndPointAngle, int laneCrownPointCenter, int laneCrownPointRight, int laneCrownPointLeft, int laneAngle, SpeedLimitList speedLimits, boolean speedLimitsExist) {
```

Comment:
I think we can remove this Boolean as we are not going to use it

Reply:
Resolved